pkg/odo/cli/registry/registry.go

```diff
@@ -257,3 +265,12 @@ func boolToYesNo(b bool) string {
 	}
 	return "N"
 }
+
+func getVersion(stack api.DevfileStack, v string) (api.DevfileStackVersion, error) {
+	for _, version := range stack.Versions {
+		if version.Version == v {
+			return version, nil
+		}
+	}
+	return api.DevfileStackVersion{}, errors.New("version not found")
```

Comment:
To help understand the error:

```suggestion
	return api.DevfileStackVersion{}, fmt.Errorf("version %q not found in Devfile stack for %q", v, stack.Name)
```